src/viewer.js

```diff
@@ -419,9 +446,16 @@ function _updateFeatureMeasurements (map, feature, pyramid) {
         })
       ]
     })
+    addOrUpdateMeasurement(feature, measurement);
+    return;
   }
+}
 
+const addOrUpdateMeasurement = (feature, measurement) => {
```

Comment:
This should probably be a private function, too.

```suggestion
const _addOrUpdateMeasurement = (feature, measurement) => {
```